.github/workflows/bump-bulk-cdk-and-release-connectors.yml

```diff
@@ -10,8 +10,93 @@ on:
         options:
           - airbytehq/airbyte
 jobs:
-  bump-cdk-version-and-merge:
+  publish-bulk-cdk:
+    uses: ./.github/workflows/publish-bulk-cdk.yml
+    secrets: inherit
+  bump-cdk-version:
     runs-on: ubuntu-24.04
+    outputs:
+      # Output the matrix config as a JSON string
+      modified_connectors: ${{ steps.export-connection-modified.outputs.modified_connectors }}
+    needs: [publish-bulk-cdk]
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+        with:
+          # Fetch all history for all tags and branches
+          fetch-depth: 0
+      - name: Fetch version from repo
+        id: fetch-version
+        run: |
+          # 1. Define the target URL
+          REPO_URL="https://airbyte.mycloudrepo.io/public/repositories/airbyte-public-jars/io/airbyte/bulk-cdk/bulk-cdk-core-load/"
```

Comment:
does that mean this only works for the bulk-load cdk?

Reply:
Yes, there is another workflow for the Java one that was merged last week.

Reply:
You mean the bulk-extract CDK?